### PR TITLE
Add project roles documentation

### PR DESCRIPTION
**Pull request by lmontopo**

### CODE REVIEW

File: source/includes/project_roles.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+Accept: application/json
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "results": [
```
Comment (sibartlett) [inline source/includes/project_roles.md]: This endpoint doesn't wrap the resource in a "results" array.

Comment (lmontopo) [inline source/includes/project_roles.md]: good catch! I'll change it.

File: source/includes/project_roles.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+Accept: application/json
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    {
```
Comment (sibartlett) [inline source/includes/project_roles.md]: You still have it nested in another object ;)

Comment (lmontopo) [inline source/includes/project_roles.md]: gah! thanks for catching this! fixing.

**Comment by lmontopo** [comment]
I think all mistakes have been fixed.  If someone wants to take a final look and then merge it while I'm away that'd be great!

**Pull request merged**